README.adoc

```diff
@@ -141,25 +147,30 @@ The `@Timed` annotation applies to the `get()` method to track how frequently th
 invoked and also how long it takes for the invocation to complete. This annotation is
 followed by fields that correspond to metadata fields:
 
-* The `name` field is optional and sets the name of the metric.
-* The `unit` field has a default value of `seconds`.
-* The `description` field is optional and describes the purpose of the metric
-annotation.
+|===
+| *Field* | *Purpose*
+|`name`  | Optional and sets the name of the metric.
+|`description` | Optional and describes the purpose of the metric annotation.
+|===
 
 The `@Counted` annotation applies to the `list()` method. This annotation counts how many times
 you request the list of systems methods by visiting the `http://localhost:9080/inventory/systems` URL to
 retrieve the information for a list of all previously registered hosts:
 
-* The `absolute` field is set to `true` to give the metric the name of the method (without the class and package names).
-* The `description` field is optional and describes the purpose of the metric annotation.
-* The `monotonic` field is set to `true` to make the counter increment only.
+|===
+| *Field* | *Purpose*
+| `absolute` | Gives the metric the name of the method (without the class and package names) if it is set to `true`.
```

Comment:
We generally try to avoid parenthesis since they indicate to the reader that the info is less important. If the info is truly not so important, why include it at all? This info looks useful, though, so I would just remove the parenthesis here.

Reply:
Done